martha_v2/martha_v2.js

```diff
@@ -11,7 +11,9 @@ function parseRequest(req) {
 
 function maybeTalkToBond(req, provider = BondProviders.default) {
     let myPromise;
-    if (req && req.headers && req.headers.authorization) {
+    // Currently HCA data access does not require additional credentials.
+    // The HCA checkout buckets allow object read access for GROUP_All_Users@firecloud.org.
+    if (req && req.headers && req.headers.authorization && provider !== BondProviders.HCA) {
```

Comment:
https://github.com/mikebaumann/martha/blob/764ec6a1bb3f39349165b113facb0c2932069297/fileSummaryV1/service_account_keys.js#L5 

We need to do something similar which is if the provider is HCA, then just return a resolved Promise.

This will ensure that behavior in both endpoints, `martha_v2` and `fileSummaryV1` remain consistent.

Reply:
This change has now been made, as per my recent comment above.